org.eclipse.xtext.common.types/src/org/eclipse/xtext/common/types/descriptions/ClasspathTypeDescriptor.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2015, 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.xtext.common.types.descriptions;
+
+import java.util.List;
+
+import org.eclipse.xtext.naming.QualifiedName;
+import org.eclipse.xtext.xbase.lib.Conversions;
+
+import com.google.common.base.Splitter;
+
+public class ClasspathTypeDescriptor implements ITypeDescriptor {
+	private static final Splitter PACKAGE_AND_NESTED_CLASS_SPLITTER = Splitter.onPattern("\\.|\\$");
+
+	private final String name;
+	private final int modifiers;
+
+	public ClasspathTypeDescriptor(String name, int modifiers) {
+		super();
```

Comment:
Unneeded

Reply:
Forgot about this one?

Reply:
Yes, seems so. Will update and merge then.